tests/sdk/sdk_integration_test.py

```
@@ -0,0 +1,106 @@
+    sessions_page = honcho_test_client.get_sessions()
+    assert len(list(sessions_page)) == 1
+
+    session.set_metadata({"bar": "baz"})
+    metadata = session.get_metadata()
+    assert metadata == {"bar": "baz"}
+
+    assistant = honcho_test_client.peer(id="assistant")
+    user = honcho_test_client.peer(id="user")
+
+    session.add_peers(
+        [assistant, (user, SessionPeerConfig(observe_others=False, observe_me=False))]
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)